cryptoservice/crypto_service_test.go

```diff
@@ -138,13 +139,16 @@ func (c CryptoServiceTester) TestSignWhenMultipleKeystores(t *testing.T) {
 	assert.NoError(t, err, c.errorMsg("error creating key"))
 
 	for _, store := range cryptoService.keyStores {
-		err := store.AddKey(privKey.ID(), "root", privKey)
```

Comment:
Non-blocking - this test seems to now actually be testing that `GetPrivateKey` returns the first key it finds that matches.  The signing part no longer seems relevant, since the previous test tests that one can sign with a private key.  Would it make sense to change this to `TestGetPrivateKeyWithMultipleKeystores` instead and skip the signing part?